src/home/rooms_sidebar.rs

```diff
@@ -113,6 +116,11 @@ live_design! {
                 text_style: <TITLE_TEXT>{}
             }
         }
+        <SearchBar> {
+            input = {
+                empty_message: "Search by room name"
```

Comment:
There are multiple criteria we can search for, right? not only the room name?
```suggestion
                empty_message: "Search for rooms..."
```